bpo-41979: Accept star-unpacking on with-item targets (#22611)
python/cpython: Misc/NEWS.d/next/Core and Builtins/2020-10-09-10-55-50.bpo-41979.ImXIk2.rst

> Accept star-unpacking on with-item targets for the PEG parser.

Reviewer: ```suggestion
Star-unpacking is now allowed for with item's targets in the PEG parser.
```
